maxim/logger/livekit/agent_session.py

```
@@ -0,0 +1,172 @@
+def intercept_session_start(self: AgentSession, *args, **kwargs):
+    """
+    This function is called when a session starts.
+    This is the point where we create a new session for Maxim.
+    The session info along with room_id, agent_id, etc is stored in the thread-local store.
+    """
+    maxim_logger = get_maxim_logger()
+    scribe().debug(
+        f"[{self.__class__.__name__}] Session started; args={args}, kwargs={kwargs}"
+    )
+    # getting the room_id
+    room_id = kwargs.get("room", None)
```

Ruff (0.11.9)

Use `kwargs.get("room")` instead of `kwargs.get("room", None)`

Replace `kwargs.get("room", None)` with `kwargs.get("room")`

(SIM910)